README.md

```diff
@@ -41,6 +41,8 @@ The required settings are listed in `settings.yaml.example`. There may be other
 * host: the host the web service runs on
 * log_level: The lowest priority log level that is logged to the output sink. Value can be one of `error`, `warn`, `info`, `debug`, or `trace`.
 * port: the port the web service runs on
+* sentry_dsn: The [DSN identifier] for the Sentry instance
+* sentry_environment: The environment passed to Sentry. Should be the same as `environment`.
```

Comment:
Writing this made me ask the question, "is this even necessary" ?

Reply:
Agree. But it's in the config that ops set up so lets just use it.